PS4k/src/com/vitco/core/EngineViewPrototype.java

```diff
@@ -86,9 +86,28 @@ public final int getWidth() {
         return container.getWidth();
     }
     // get the image rendered in container in high quality
-    public BufferedImage getImage() {
+    public BufferedImage getImage() throws Exception {
         return container.getImage();
     }
+
+    // get camera position
+    public final float[] getCamPosition() { return this.camera.getCamPosition(); }
+
+    public BufferedImage getImage(final Color bgColor, final boolean addWatermark) throws Exception { return container.getImage(bgColor, addWatermark); }
```

Comment:
just an agreement that i will never modify it, will change it if you don't like it